src/app/vault/password-generator.component.html

```diff
@@ -27,13 +27,17 @@
                         </button>
                     </div>
                     <div class="box-content condensed" [hidden]="!showOptions">
-                        <div class="box-content-row" appBoxRow>
-                            <label for="type" class="sr-only">{{'type' | i18n}}</label>
-                            <select id="type" [(ngModel)]="options.type" (change)="saveOptions()">
-                                <option value="password">{{'password' | i18n}}</option>
-                                <option value="passphrase">{{'passphrase' | i18n}}</option>
-                            </select>
-                        </div>
+                        <div class="box-content-row box-content-row-radio">
```

Comment:
I think this element is missing the closing `</div>` tag.

Reply:
It was. Corrected.